VaultOfTheIncarnates/Locales/koKR.lua

```diff
@@ -1,65 +1,65 @@
 local L = BigWigs:NewBossLocale("Eranog", "koKR")
 if not L then return end
 if L then
-	-- L.custom_on_nameplate_fixate = "Fixate Nameplate Icon"
-	-- L.custom_on_nameplate_fixate_desc = "Show an icon on the nameplate of Frenzied Tarasek that are fixed on you.\n\nRequires the use of Enemy Nameplates and a supported nameplate addon (KuiNameplates, Plater)."
-
-	-- L.molten_cleave = "Frontal"
-	-- L.incinerating_roar = "Roar"
-	-- L.molten_spikes = "Spikes"
-	-- L.collapsing_army = "Army"
-	-- L.greater_flamerift = "Mythic Add"
-	-- L.leaping_flames = "Flames"
+	 L.custom_on_nameplate_fixate = "시선 고정 네임플레이트 아이콘"
+	 L.custom_on_nameplate_fixate_desc = "당신에게 시선이 고정된 타라세크의 네임플레이트에 아이콘을 표시합니다.\n\n 네임플레이트를 지원하는 애드온(KuiNameplates, Plater)을 사용하고 적 네임플레이트가 활성화된 상태여야함."
+
+	 L.molten_cleave = "전방기"
+	 L.incinerating_roar = "포효"
+	 L.molten_spikes = "쐐기"
+	 L.collapsing_army = "쫄 소환"
+	 L.greater_flamerift = "신화 쫄"
+	 L.leaping_flames = "Flames"
 end
 
 L = BigWigs:NewBossLocale("Terros", "koKR")
 if L then
-	-- L.rock_blast = "Soak"
-	-- L.resonating_annihilation = "Annihilation"
-	-- L.awakened_earth = "Pillar"
-	-- L.shattering_impact = "Slam"
-	-- L.concussive_slam = "Tank Line"
+	 L.rock_blast = "같이 맞아주기"
+	 L.resonating_annihilation = "절멸 - 잔해 조심"
+	 L.awakened_earth = "기둥 생성"
+	 L.shattering_impact = "충격 - 잔해 조심"
+	 L.concussive_slam = "탱커 조심"
 	-- L.infused_fallout = "Dust"
 end
 
 L = BigWigs:NewBossLocale("The Primal Council", "koKR")
 if L then
-	-- L.primal_blizzard = "Blizzard" -- Primal Blizzard
-	-- L.earthen_pillars = "Pillars" -- Earthen Pillars
-	-- L.meteor_axes = "Axes" -- Meteor Axes
-	-- L.meteor_axe = "Axe" -- Singular
-	-- L.meteor_axes_melee = "Melee Axe"
-	-- L.meteor_axes_ranged = "Ranged Axe"
-	-- L.conductive_marks = "Marks" -- Conductive Marks
-	-- L.conductive_mark = "Mark" -- Singular
-
-	-- L.custom_off_chain_lightning = "Chain Lightning is off by default, enable this to enable chain lightning."
-
-	-- L.custom_on_stop_timers = "Always show ability bars"
-	-- L.custom_on_stop_timers_desc = "Abilities that will always be shown: Conductive Mark"
+	 L.primal_blizzard = "눈보라" -- Primal Blizzard
+	 L.earthen_pillars = "기둥" -- Earthen Pillars
+	 L.meteor_axes = "도끼" -- Meteor Axes
+	 L.meteor_axe = "도끼" -- Singular
+	 L.meteor_axes_melee = "근딜 도끼"
+	 L.meteor_axes_ranged = "원딜 도끼"
+	 L.conductive_marks = "표식" -- Conductive Marks
+	 L.conductive_mark = "표식" -- Singular
+
+	 L.custom_off_chain_lightning = "연쇄 번개는 기본적으로 꺼져있습니다. 활성화해서 연쇄 번개를 활성화하십시오."
+
+	 L.custom_on_stop_timers = "항상 능력 바 표시Always show ability bars"
```

Comment:
did you mean to leave the english in?

Reply:
yes, I could translate it but I want to wait for it to go live before I do it

Reply:
for the other abilities I can look up stuff on wowhead and whatnot but I couldn't really grasp what those really meant